events/elections/2021/README.md

```diff
@@ -0,0 +1,250 @@
+
+Examples of contributions that would be considered:
+* Slack admins who are not active in GitHub
+* Code of Conduct Committee members whose actions are private by default
+
+Examples of contributions that would NOT be considered:
+* Contributions to ecosystem projects and products
+* Organizing meetups or podcasts
+
+### Schedule
+
+<!-- While finalizing the dates in the schedule, ensure that:
+- The Steering Committee and candidate Q+A occurs at a public SC meeting
+  (usually a Monday).
+- Dealine to submit voter exception forms and request a
+  replacement ballot is ~3 days before voting closes.
+- Private announcement of results to SC members is at least ~2 days
+  before private announcement to all candidates.
+- The interval between private announcement to all candidates and the
+  public announcement is a weekend.
+-->
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| July 1       | Steering Committee selects Election Committee |
+| August XX    | Announcement of Election and publication of voters.md |
+| August XX    | Steering Committee Meeting with Q+A with the candidates and community |
+| September XX | All candidate bios due by 0000 UTC (5pm PST) |
+| ~1 week      | Election prep week (voters.md validation and CIVS setup and testing)
+| September XX | Election Begins via email ballots |
+| October XX   | Deadline to submit voter exception forms and request a replacement ballot |
+| October XX   | Election Closes by 0000 UTC (5pm PST) |
+| October XX   | Private announcement of Results to SC members not up for election |
+| October XX   | Private announcement of Results to all candidates |
+| October XX   | Public announcement of Results at Public Steering Committee Meeting |
+| October XX   | Election Retro |
+
+## Candidacy Process
+
+**Nomination**
+
+If you want to stand for the election, create an issue in this GitHub repo
+(kubernetes/community) with the title `Steering Committee Nomination: Your Name (@yourgithub)`.
+After creating the issue, send an email to kubernetes-dev@googlegroups.com
+with a link to the issue. The subject line of the email should be same as
+the title of the issue.
+
+If you want to nominate someone else, you may do so, but PLEASE talk to them
+first.
+
+If you wish to accept a nomination from someone else, reply to the nomination
+**issue** saying something like "I accept the nomination".
+
+**Endorsement**
+
+Once nominated, you must get the endorsement of three (3) different eligible
+voters from three (3) different employers.  If you are eligible to vote
+yourself, you count as one of the three. Endorsements from non-voting members
+does not count towards the final count.
+
+[Eligible voters] may endorse candidates of their choosing by replying to the
+candidate's nomination **issue** saying something like "I endorse this nominee,
+and I work for <COMPANY>" or "+1". Please state that you are in voters.md,
```

Comment:
```suggestion
and I work for <COMPANY>" or "+1". Please state that you are an eligible voter,
```

voters.md as part of the process is going away next year, so I'm just getting a head-start on removing references to it.